File: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples-js/ServerExtendedAuditingSettingsList.js
const { SqlManagementClient } = require("@azure/arm-sql");
const { DefaultAzureCredential } = require("@azure/identity");

/**
 * This sample demonstrates how to Lists extended auditing settings of a server.
 *
 * @summary Lists extended auditing settings of a server.
 * x-ms-original-file: specification/sql/resource-manager/Microsoft.Sql/preview/2020-11-01-preview/examples/ServerExtendedAuditingSettingsList.json
 */
async function listExtendedAuditingSettingsOfAServer() {
  const subscriptionId = "00000000-1111-2222-3333-444444444444";
  const resourceGroupName = "blobauditingtest-4799";
  const serverName = "blobauditingtest-6440";
  const credential = new DefaultAzureCredential();
  const client = new SqlManagementClient(credential, subscriptionId);
  const resArray = new Array();
  for await (let item of client.extendedServerBlobAuditingPolicies.listByServer(
    resourceGroupName,
    serverName
  )) {
    resArray.push(item);
  }
  console.log(resArray);
}

listExtendedAuditingSettingsOfAServer().catch(console.error);

File: specification/sql/resource-manager/Microsoft.Sql/SQL/stable/2025-01-01/examples-java/ServerExtendedAuditingSettingsList.java

/**
 * Samples for ExtendedServerBlobAuditingPolicies ListByServer.
 */
public final class Main {
    /*
     * x-ms-original-file: 2025-01-01/ServerExtendedAuditingSettingsList.json
     */
    /**
     * Sample code: List extended auditing settings of a server.
     * 
     * @param manager Entry point to SqlServerManager.
     */
    public static void listExtendedAuditingSettingsOfAServer(com.azure.resourcemanager.sql.SqlServerManager manager) {
        manager.serviceClient().getExtendedServerBlobAuditingPolicies().listByServer("blobauditingtest-4799",
            "blobauditingtest-6440", com.azure.core.util.Context.NONE);
    }
}
